backend/routes/restaurant.js

```diff
@@ -6,6 +6,7 @@ import connection from '../database';
 const router = express.Router();
 router.use(bodyParser.urlencoded({ extended: true }));
 
+// Gets restaurant details, used by front end to restaurant id,name and banner image url(todo) to end user.
```

Comment:
should it be 'used by front end to _**provide**_ restaurant id, name and banner image to end user'?

Reply:
English is my second language. Apologies and fixed.